src/applications/simple-forms/21-0845/definitions/form-address.js

```diff
@@ -0,0 +1,338 @@
+// copy of src/platform/forms/definitions/address.js,
```

Comment:
this is same as 21-10210 right? could also move this to simple-forms/shared/definitions.

Reply:
Refactored & moved to simple-forms/shared/definitions.